src/main/java/Event/Event.java

```diff
@@ -0,0 +1,21 @@
+package Event;
+
+public class Event extends Task {
+    protected String from;
+    protected String to;
+    public Event(String input) {
+
+        super(input);
+        String[] firstPart = input.split(" /from ", 2);
+        this.description = firstPart[0].substring(6).trim(); // Remove "event" command and trim spaces.
+        String[] secondPart = firstPart[1].split(" /to ", 2);
+        this.from = secondPart[0];
+        this.to = secondPart[1];
+        this.eventType = "[E]";
+
+    }
+    @Override
```

Comment:
You could consider using consistent spacing throughout the code base (between methods, within methods, etc.)